### PR TITLE
Create employees class

### LINKED ISSUE #6: Create a Portion for Employee
Create a portion for employee with:

`Username`, `Enrollment`, `FullName`, `CompanyName`, `Role`

### PR DESCRIPTION
**Pull request by leisiamedeiros**
Resolve #6

### CODE REVIEW

**Comment by leisiamedeiros** [comment]
@yanjustino please review and fell free to request changes if it needed

File: Polyjuice/Potions/Employee.cs [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    /// <summary>
+    /// An employee that works for an organization
+    /// </summary>
+    public static class Employee
+    {
+        public static string Username => Name.FirstName;
+        public static string FullName => $"{Username} {Name.LastName}";
+        public static string Enrollment => "########".Numerify();
+        public static string Company => GlobalCompany.CompanyName;
+        public static string Role => $"{Read("Job", "JobNouns").Rand()}";
```
Comment (yanjustino) [inline Polyjuice/Potions/Employee.cs]: Use class [`Job`](https://github.com/yanjustino/polyjuice/blob/master/Polyjuice/Potions/Job.cs) to specify the Role attribute

```suggestion
        public static string Role => Job.Title;
```
If this suggestion is not applicable, you could implement a new attribute in `Job` class, such as:

```c#
   public static string SummaryTitle =>    $"{Read("Job", "JobNouns").Rand()} " ;
```

**Review by yanjustino — COMMENTED** [review]
Codigo para review

File: Polyjuice/Potions/Employee.cs (hunk 1 above)
Comment (leisiamedeiros) [reply Polyjuice/Potions/Employee.cs]: @yanjustino Accepted the alternative suggestion

**Pull request merged**